src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -82,14 +65,20 @@ def __init__(self, ensemble: Ensemble, config: EvaluatorServerConfig):
         self._max_batch_size: int = 500
         self._batching_interval: float = 2.0
         self._complete_batch: asyncio.Event = asyncio.Event()
+        self._server_started: asyncio.Event = asyncio.Event()
+        self._clients_connected: set[bytes] = set()
+        self._clients_empty: asyncio.Event = asyncio.Event()
```

Comment:
The benefit of having it as an event, that we can do `asyncio.wait_for(self._clients_empty.wait(), timeout=...)`. Otherwise, before closing evaluator I'd need to while loop and do the timeout manually.